src/index.html

```diff
@@ -28,7 +28,7 @@
         </div>
         <header class="header-img">
             <img alt="leftcloudimg" class="left-cloud" src="../Assets/HTML&CSS/GeneralImages&Icons/cloud-8121.png">
-            <img alt="lightning" class="lightning" src="../Assets/HTML&CSS/lightning/pngwing.com (3).png">
```

Comment:
Why is the middle sections tiles not interacting properly on hovering? Also horizontal scroll seems to not work properly

Reply:
The horizontal scroll still doesn't seem to be working properly. Not by clicking on the scroll arrow button but by mouse scroll(hold shift + scroll).

Reply:
Its working for me Sai